MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+### DataFormCheckBoxItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[CheckedColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormCheckBoxItem.html#Syncfusion_XForms_DataForm_DataFormCheckBoxItem_CheckedColor)'| markdownify }}</td>
+<td>Color</td>
+<td>Gets or sets the color of CheckBox editor.</td>
+</tr>
+</table>
+
+### DataFormSwitchItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>Nil</td>
+<td>OnColor</td>
+<td>Gets or sets the color of the switch when it is in the on state.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ThumbColor</td>
+<td>Gets or sets the thumb color of the switch editor.</td>
+</tr>
+</table>
+
+### DataFormTextEditorItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[KeyBoard](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTextItemBase.html#Syncfusion_XForms_DataForm_DataFormTextItemBase_KeyBoard)'| markdownify }}</td>
+<td>Keyboard</td>
+<td>Gets or sets the type of keyboard used to display in the text, multiline and password editor.</td>
+</tr>
+</table>
+
+### DataFormDateItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[Format](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormDateItem.html#Syncfusion_XForms_DataForm_DataFormDateItem_Format)'| markdownify }}</td>
+<td>Format</td>
+<td>Gets or sets the time format for time picker.</td>
+</tr>
+
+<tr>
+<td>{{'[MaximumDate](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormDateItem.html#Syncfusion_XForms_DataForm_DataFormDateItem_MaximumDate)'| markdownify }}</td>
+<td>MaximumDisplayDate</td>
+<td>Gets or sets the maximun display date to restrict the visible dates of the date editor.</td>
+</tr>
+
+<tr>
+<td>{{'[MinimumDate](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormDateItem.html#Syncfusion_XForms_DataForm_DataFormDateItem_MinimumDate)'| markdownify }}</td>
+<td>MinimumDisplayDate</td>
+<td>Gets or sets the minimum display date to restrict the visible dates of the date editor.</td>
+</tr>
+</table>
+
+### DataFormTimeItem
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[Format](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTimeItem.html#Syncfusion_XForms_DataForm_DataFormTimeItem_Format)'| markdownify }}</td>
+<td>Format</td>
+<td>Gets or sets the time format for Time Picker.</td>
+</tr>
+</table>
+
+### DataFormListItem
+
+The common properties of `Xamarin SfDataForm` list items (` DataFormPickerItem`, ` DataFormAutoCompleteItem`, ` DataFormDropDownItem` and ` DataFormRadioGroupItem` ) are added to [DataFormListItem] class in `.NET MAUI SfDataForm` 
```

Comment:
The common properties of the `Xamarin SfDataForm` list items (` DataFormPickerItem,` ` DataFormAutoCompleteItem,` ` DataFormDropDownItem,` and ` DataFormRadioGroupItem` ) are added to the [DataFormListItem] class in `.NET MAUI SfDataForm.`